### DISCUSSION TITLE
VanJs Bootstrap Components

### DISCUSSION THREAD

**Discussion by WilliCommer** [vanjs-org/van#235] [Show and tell]
if anyone likes bootstrap, they should take a look at my vanjs-bootstrap library. The demo is built entirely with VanJs.

- [demo](https://familiecommer.de/vanjs-bootstrap-demo/)
- [github](https://github.com/WilliCommer/vanjs-bootstrap)
- [npm](https://www.npmjs.com/package/vanjs-bootstrap)

**Comment by Tao-VanJS** [comment]
Thanks for sharing!

I added your library into VanJS's [Community Add-ons](https://vanjs.org/#community-add-ons) section.
